tools/python/boutiques/bosh.py

```diff
@@ -141,9 +144,9 @@ def execute(*params):
             raise SystemExit("Input file {} does not exist.".format(inp))
         if inp and not inp.endswith(".json"):
             raise SystemExit("Input file {} must end in 'json'.".format(inp))
-        if not os.path.isfile(descriptor):
-            raise SystemExit("JSON descriptor {} does not seem to exist."
-                             .format(descriptor))
+        # if not os.path.isfile(descriptor):
+        #     raise SystemExit("JSON descriptor {} does not seem to exist."
+        #                      .format(descriptor))
```

Comment:
you might as well just remove these lines

Reply:
oops, forgot to remove that